.github/workflows/integration-tests.yml

```diff
@@ -324,7 +333,7 @@ jobs:
 
       - name: Clear cache
         run: |
-          rm -rf ~/.triton
+          rm -rf ~/.triton/cache
```

Comment:
Does this directory exist anymore?  I don't see it on my machine.  On my machine we seem to be writing kernels to /tmp/triton instead.

Reply:
Are you using AMD or NV gpus? Thanks

Reply:
I'm using NV GPUs.  If AMD puts it in a different place we should fix that.

Reply:
`~/.triton/cache` is the default. @jlebar it is different for you because your VM overrides it `TRITON_CACHE_DIR` :)